Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+# Other Formatting Options
+
+Improve the presentation and structure of your content by incorporating special page elements.
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
+* A row with cells that contain only dashes (`-`); three or more for each 
+column.
+* One or more rows of content.
```

Comment:
I would not recommend using a table for a single row of content. A table should have at least two rows of content, and a minimum of two columns.